### DISCUSSION TITLE
How can I permanently disable a module?

### DISCUSSION THREAD

**Discussion by geoidesic** [zotonic/zotonic#3919] [Q&A] [answered]
I've disabled the comments module several times... but it keeps re-enabling itself. I don't know what's causing that? Perhaps rebuilding docker and restarting the zotonic service? Maybe it's the Rescan Modules function that causes it? Either way it's super annoying :) 

Is there a way to make it stick?

**Comment by mworrell** [comment, marked as answer]
It is probably in the `priv/zotonic_site.config` file.  Check the `modules` list and remove `mod_comment`
